src/main/kotlin/com/exactpro/th2/codec/EventProcessor.kt

```diff
@@ -0,0 +1,209 @@
+/*
+ *  Copyright 2020-2023 Exactpro (Exactpro Systems Limited)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.exactpro.th2.codec
+
+import com.exactpro.th2.codec.api.impl.ReportingContext
+import com.exactpro.th2.common.event.Event
+import com.exactpro.th2.common.event.Event.Status
+import com.exactpro.th2.common.event.Event.Status.PASSED
+import com.exactpro.th2.common.event.Event.Status.FAILED
+import com.exactpro.th2.common.event.EventUtils
+import com.exactpro.th2.common.event.IBodyData
+import com.exactpro.th2.common.grpc.EventID
+import com.exactpro.th2.common.grpc.MessageID
+import mu.KotlinLogging
+
+abstract class AbstractEventProcessor {
+    fun onEvent(message: String, messagesIds: List<MessageID> = emptyList(), body: List<String> = emptyList()): String {
+        LOGGER.warn { "$message. Messages: ${messagesIds.joinToReadableString()}" }
```

Comment:
Please use com.exactpro.th2.common.utils.message.logId extension property from common-utils instead of custom method